src/submitSite.js

```diff
@@ -0,0 +1,197 @@
+import { Builder, By, Key, until } from 'selenium-webdriver';
+
+(async function example() {
+  const dataUser = {
+    "userTitle":"Mr",
+    "userFirstName":"Glenn",
+    "userLastName":"Frogg",
+    "userAddressLine1":"101 1st Street",
+    "userAddressLine2":"202 2nd Street",
+    "userAddressCountry":"Canada",
+    "userAddressCityMunicipality":"Waterloo",
+    "userAddressProvinceTerritory":"Ontario",
+    "userAddressCode":"N1N 2H2",
+    "userContactNumber":"819-555-1234",
+    "userContactEmail":"janeandjoe@name.com",
+    "userContactConfirmEmail":"janeandjoe@name.com",
+    "userContactLanguage":"english",
+    "userSpammedNumber":"819-555-1234",
+    "userSpammedPersonal":true,
+    "userSpammedBusiness":false,
+    "userSpammedProvider":"Rogers"
+  }
+  const dataCall = {
+    "callUserAction":"Answered",
+    "callOnPickup":"live",
+    "callNameOrNumber":"number",
+    "callDeclaration":"yes",
+    "callerNumber":"519-635-6921",
+    "callerID":"",
+    "callerAddress":"",
+    "callerWebsite":"",
+    "callDate":"2021-10-29",
+    "callTime":"12:45",
+    "reasonSell":true,
+    "reasonWon":false,
+    "reasonSurvey":false,
+    "reasonPolitical":false,
+    "reasonDonation":false,
+    "reasonDebt":false,
+    "reasonPsa":false,
+    "reasonUnknown":false,
+    "purchaseStatus":"no",
+    "purchaseNow":false,
+    "purchaseRecent":false,
+    "purchaseDistant":false,
+    "businessInquiry":true,
+    "businessContract":false,
+    "businessDivulge":true,
+    "businessVisit":false,
+    "ceaseNow":false,
+    "ceasePrior":false,
+    "ceaseRefuse":false,
+    "consent":true
+  }
+  let driver = await new Builder().forBrowser('chrome').build();
+  try {
+    // Navigate to Url
+    await driver.get('https://lnnte-dncl.gc.ca/en/Consumer/Complaint/#!/');
+    if (dataCall.callUserAction==="Answered"){
+      await driver.findElement(By.id('radio-answer-phone')).click();
+    }
+    await driver.findElement(By.css('div.submit-container button')).click();
+    
+    // 2nd page #Complaint details
+    if (dataCall.callOnPickup==="live"){
+      await driver.findElement(By.id('radio-phone-live')).click();
+    }
+    if (dataCall.callNameOrNumber==="number"){
+      await driver.findElement(By.id('radio-comp-phone')).click();
+    }
+    if (dataCall.callDeclaration==="yes"){
+      await driver.findElement(By.id('radio-declar-1')).click();
+    }
+    await driver.findElement(By.css('div.submit-container button:nth-child(2)')).click();
+    
+    // 3rd Page #Caller details
+    await driver.findElement(By.id('number')).sendKeys(dataCall.callerNumber);
+    await driver.findElement(By.id('callerId')).sendKeys(dataCall.callerID);
+    await driver.findElement(By.id('address')).sendKeys(dataCall.callerAddress);
+    await driver.findElement(By.id('website')).sendKeys(dataCall.callerWebsite);
+    await driver.findElement(By.xpath('//*[@id="complaint-3"]/div[3]/button[2]')).click();
```

Comment:
One thing I see our SDETs do is take these ids/xpaths etc. and turn them into constants at the top of the file so that's a little easier to read

Reply:
Hmmm, alright I'll do that with the xpaths at least, cause they are not readable at all. The ids here seem readable to me, but of course I wrote it. Do you think I should change the ids as well?